scripts/serve.ts

```diff
@@ -287,6 +337,20 @@ async function createGateway(name: string) {
   }
 }
 
+function getProverFactory(chain: Chain): LatestProverFactory<AbstractProver> {
+  switch (chain) {
+    case CHAINS.ZKSYNC:
+    case CHAINS.ZKSYNC_SEPOLIA:
+      return ZKSyncProver;
+    // NOTE: linea should use eth_getProof instead of linea_getProof
```

Comment:
Could you give some additional color to this comment?

Reply:
linea rollup state is a SMT w/Mimc root using `linea_getProof`
linea chain state is a PMT w/Keccak root using `eth_getProof`